python/selfie-lib/selfie_lib/Literals.py

```diff
@@ -71,6 +72,174 @@ def parse(self, string: str, language: Language) -> int:
         return int(string.replace("_", ""))
 
 
+TRIPLE_QUOTE = '"""'
+
+
+class LiteralString(LiteralFormat[str]):
+    def encode(
+        self, value: str, language: Language, encoding_policy: EscapeLeadingWhitespace
+    ) -> str:
+        if "/n" not in value:
+            if language == Language.PYTHON:
+                return self._encodeSinglePython(value)
+        else:
+            if language == Language.PYTHON:
+                return self.encodeMultiPython(value, encoding_policy)
+
+    def parse(self, string: str, language: Language) -> str:
+        if not string.startswith(TRIPLE_QUOTE):
+            if language == Language.PYTHON:
+                return self._parseSinglePython(string)
+        else:
+            if language == Language.PYTHON:
+                return self.parseMultiPython(string)
```

Comment:
I know `Language.PYTHON` is the only value right now, but what would happen if the language was something else?

I honestly don't know, I'm guessing it would return `None`? Regardless, there should be an `else: throw NotImplemented` or something like that. Probably makes sense to make the `if language == PYTHON` be the outer if.